grpc/server.go

```diff
@@ -0,0 +1,84 @@
+package sentrygrpc
+
+import (
+	"context"
+
+	"github.com/getsentry/sentry-go"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// UnaryServerInterceptor is a grpc interceptor that reports errors and panics
+// to sentry. It also sets *sentry.Hub to context.
+func UnaryServerInterceptor(opts UnaryServerInterceptorOptions) grpc.UnaryServerInterceptor {
+	if opts.ReportOn == nil {
+		opts.ReportOn = ReportAlways
+	}
+
+	return func(
+		ctx context.Context,
+		req interface{},
+		info *grpc.UnaryServerInfo,
+		handler grpc.UnaryHandler,
+	) (_ interface{}, err error) {
```

Comment:
Either name all return values or none. I glanced over other unary interceptor implementations, I think the common case is the former:

```suggestion
	) (resp interface{}, err error) {
```

Reply:
Sounds reasonable. I'll fix 👍